builder/ucloud/common/client.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/ucloud/ucloud-sdk-go/services/unet"
 	"github.com/ucloud/ucloud-sdk-go/services/vpc"
 	"github.com/ucloud/ucloud-sdk-go/ucloud"
-	"github.com/ucloud/ucloud-sdk-go/ucloud/error"
+	uerr "github.com/ucloud/ucloud-sdk-go/ucloud/error"
```

Comment:
The package inside `github.com/ucloud/ucloud-sdk-go/ucloud/error` is called `uerr` so if we don't name it in the import it will not be `error` but `uerr`. My `goimports` run 'fixed' this so that it is less confusing.